changelog/private.dd

```diff
@@ -0,0 +1,9 @@
+Deprecate the use of private selectively imported members
+
+The spec highlights clearly that a private member is visible
+only from within the same module. Due to a bug, selectively
+imported private members were visible from other modules. This
+represents a severe violation of the private keyword specification.
+At this point the use of selectively imported private members has
+been deprecated and should be transformed into an error as soon
```

Comment:
Will be transformed?

Reply:
Then again the wording in general could be improved.

Reply:
Proposed wordsmithing:

> $(LINK2 $(ROOT_DIR)spec/attribute.html#visibility_attributes, The specification states) that a private member is visible on from within the same module.  Prior to this release, due to a bug, some imported private members were visible from other modules, violating the specification.  Beginning with this release, accessing said private members from another module will result in a deprecation message which has been scheduled to be changed to an error in approximately 1 year.

Reply:
Thanks. I copy-pasted your words.